pkg/apis/externaldns/types.go

```diff
@@ -201,6 +203,7 @@ func (cfg *Config) ParseFlags(args []string) error {
 	app.Flag("txt-prefix", "When using the TXT registry, a custom string that's prefixed to each ownership DNS record (optional)").Default(defaultConfig.TXTPrefix).StringVar(&cfg.TXTPrefix)
 
 	// Flags related to the main control loop
+	app.Flag("cache-interval", "The interval between cache synchronizations in duration format (default: 1h)").Default(defaultConfig.CacheInterval.String()).DurationVar(&cfg.CacheInterval)
```

Comment:
Since the flag is only relevant for the `txt`-registry I would prefer to call it `--txt-cache-interval` similar to `txt-owner-id` and `txt-prefix`.

Reply:
This would also change the env var name and the flag definition would be better suited next to the other `txt-*`  flags.

Reply:
updated :)